octoprint_slic3r/profile.py

```diff
@@ -38,148 +38,170 @@ class SeamPositions(object):
+    avoid_crossing_perimeters = 0, 			# 0 (unchecked) or 1 (checked)
+    thin_walls = 1, 					# 0 (unchecked) or 1 (checked)
+    overhangs = 0,
+
+    support_material = 0, 				# 0 (unchecked) or 1 (checked)
+    support_material_threshold = 0,
+    support_material_pattern = SupportPatterns.HONEYCOMB,
+    support_material_spacing = 2.5,
+    support_material_angle = 0,
+    support_material_interface_layers = 3,
+    support_material_interface_spacing = 0,
+    raft_layers = 0,
+    support_material_enforce_layers = 0,
+    dont_support_bridges = 0, 				# 0 (unchecked) or 1 (checked)
+
+    retract_length = 1,
+    retract_speed = 30,
+    retract_restart_extra = 0,
+    retract_before_travel = 2,
+    retract_lift = 0,
+    retract_layer_change = 1,
+    wipe = 0,
+
+    retract_length_toolchange = 1,
+    retract_restart_extra_toolchange = 1,
+
+    cooling = 0, 					# 0 (unchecked) or 1 (checked)
+    min_fan_speed = "35",
+    max_fan_speed = "100",
+    bridge_fan_speed = "100",
+    fan_below_layer_time = 60,
+    slowdown_below_layer_time = 30,
+    min_print_speed = 10,
+    disable_fan_first_layers = 1,
+    fan_always_on = 0,					# 0 (unchecked) or 1 (checked)
+
+    skirts = 1,
+    skirt_distance = 6,
+    skirt_height = 1,
+    min_skirt_length = 0,
+    brim_width = 0,
+
+    duplicate_distance = 6,
+    xy_size_compensation = 0,
+
+    complete_objects = 0,
+    extruder_clearance_radius = 20,
+    extruder_clearance_height = 20,
+
+    notes = "",
+    resolution = 0,
+
+    extrusion_width = "100%",
+    first_layer_extrusion_width = "100%",
+    perimeter_extrusion_width = "100%",
+    external_perimeter_extrusion_width = "100%",
+    infill_extrusion_width = "100%",
+    solid_infill_extrusion_width = "100%",
+    top_infill_extrusion_width = "100%",
+    support_material_extrusion_width = "100%",
+    bridge_flow_ratio = 1,
+
+    extruder_offset = "0x0",
+    perimeter_extruder = 1,
+    infill_extruder = 1,
+    support_material_extruder = 1,
+    support_material_interface_extruder = 1,
+    ooze_prevention = 0, 				# 0 (unchecked) or 1 (checked)
+    standby_temperature_delta = -5,
+
+    #Parameters that weren't in the list that have been added
+    extrusion_axis = "E",
+    bed_shape = "0x0,100x0,100x100,0x100",
+    bed_size = (200,200),
+    before_layer_gcode = None,
+    external_fill_pattern = FillPatterns.RECTILINEAR,
+    filament_colour = "#FFFFFF",
+    infill_overlap = "15%",
+    max_print_speed = 80,
+    max_volumetric_speed = 0,
+    pressure_advance = 0,
+    solid_infill_extruder = 1,
+    support_material_contact_distance = 0.2,
+    use_volumetric_e = 0,  				# 0 (unchecked) or 1 (checked)
+    threads = 2,
+    first_layer_bed_temperature = 90,
+    interface_shells = 0, 				# 0 (unchecked) or 1 (checked)
+    post_processing = None,
+    solid_fill_pattern = FillPatterns.RECTILINEAR,
+    g0 = 0,						# 0 (unchecked) or 1 (checked)
+    duplicate = 1,
```

Comment:
And also `duplicate`, `duplicate_grid`, `scale` and `rotate`.

`solid_layers` is still missing though.